### PR TITLE
Implement simple drag functionality, setup interactjs lib

### PR DESCRIPTION
**Pull request by spaceofmiah**
This is just a simple drag functionality implemented on a styled paragraph.
Also setup **interactjs** library and included Nigeria's map

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,55 @@
...
+
+
+	<!-- 
+		*
+		*
+			Programming Script
+		*
+		*
+	-->
+	<script>
```
Comment (jamesoyanna) [inline index.html:30]: We may need to move all the code on the script to the app.js file just to have that separation of cocern

**Review by jamesoyanna — COMMENTED** [review]
So we can have all the script code on the app.js file

**Comment by jamesoyanna** [comment]
Your request has been approved.

File: index.html (hunk 1 above)
Comment (spaceofmiah) [reply index.html]: work in progress

**Pull request merged**